Sofa/GL/CMakeLists.txt

```
@@ -1,6 +1,11 @@
 cmake_minimum_required(VERSION 3.22)
 project(Sofa.GL LANGUAGES CXX)
 
+# Detect if component is compiled outside SOFA
+if ("${CMAKE_PROJECT_NAME}" STREQUAL "${PROJECT_NAME}")
+    sofa_find_package(Sofa.Config REQUIRED)
+endif()
```

`sofa_find_package` is not known before you do the real `find_package`... so this will not work.

```suggestion
if ("${CMAKE_PROJECT_NAME}" STREQUAL "${PROJECT_NAME}")
   find_package(Sofa.Config REQUIRED)
endif()
```